### DISCUSSION TITLE
Is a fabric 1.18.1 version planned?

### DISCUSSION THREAD

**Discussion by maxutka99** [fnuecke/oc2#123] [Q&A]
Is a fabric 1.18.1 version planned?

**Comment by fnuecke** [comment]
I don't intend to do it myself. There was a [start for it a while back](https://github.com/fnuecke/oc2/pull/44), but I think the person who started it doesn't have time to work on it currently.

**Reply by lucsoft** [reply]
Yeah I have my finals in less then two months

After that I could work on it again. My plan is to use Architectury this will also allow to make it for Fabric and Forge with the same codebase

**Reply by suconakh** [reply]
@lucsoft good luck!
